portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+import java.util.Date;//will use later
+public class Comment{
```

Comment:
nit: Extra space between Comment and "{".
Given here you use bracket after character, you may want to keep the functions consistent like this as well.
For example:
public Comment(...) {
...
}